src/components/textarea/textarea.component.tsx

```diff
@@ -1,8 +1,9 @@
 import React, { useRef, useEffect, useContext } from "react";
-import PropTypes from "prop-types";
-import styledSystemPropTypes from "@styled-system/prop-types";
+import { MarginProps } from "styled-system";
 
-import { InputPresentation } from "../../__internal__/input";
+import { IconType } from "components/icon";
```

Comment:
```suggestion
import { IconType } from "../icon";

Is there a reason we're not using the relative import style for this? I've seen imports break mysteriously in the past when doing it the absolute way, and in any case I prefer my suggestion for consistency if nothing else!
```

Reply:
Probably a case of vscode automatically creating the import. Will correct